web/src/pages/transaction/add.vue

```diff
@@ -3,12 +3,14 @@
 const store = ref("")
 const amount = ref(0)
 const date = ref(Date.now())
+const transactionType = ref('expense');
 
 type transaction = {
   Key: number,
   Store: string,
   Amount: number,
   Transaction_Date: number | string;
+  Type: string;
```

Comment:
I think it might make more sense for the number being negative or positive determining whether the transaction is an income or expense? You could just multiply the value by `-1` on form submit if Income is selected?